src/style.css

```diff
@@ -0,0 +1,120 @@
+:root {
+  --my-text-color: black;
+  --my-bg-color: bisque;
+  --my-gray-color: rgb(236, 233, 233);
+
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 12px;
+  font-weight: 100;
+}
+
+.flex {
+  display: flex;
+  flex-direction: row;
+}
+
+.flex-columns {
+  display: flex;
+  flex-direction: column;
+}
+
+.my-bg-gray {
+  background-color: var(--my-gray-color);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: whitesmoke;
+}
+
+/* MAIN CONTAINER */
+
+.main-container {
+  width: 50%;
+  margin: 5% auto;
+  border: 1px solid var(--my-gray-color);
+  background-color: white;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: 0;
+  justify-content: space-between;
+  padding: 1%;
+}
+
+.row {
+  border-top: 1px solid var(--my-gray-color);
+  border-bottom: 1px solid var(--my-gray-color);
+}
+
+/* Title DIV */
+#title {
+  justify-content: space-between;
+}
+
+#title h1 {
+  font-size: 1rem;
+  margin-bottom: 0;
+  padding: 1%;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  background-color: var(--my-gray-color);
+}
+
+#title-image {
+  width: 5%;
+  background: url(images/circle.jpg) no-repeat;
+  background-size: 60%;
+}
+
+/* Form DIV */
+input {
+  width: 100%;
+  border: 0;
+}
```

Comment:
[OPTIONAL] Well done overall 💯 but consider improving your styles according to [this video](https://youtu.be/AcUd-_Yjjqg)

Reply:
Well received @didierganthier , thank you, I surely will.